AirCasting/AirCastingRelease.entitlements

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file?

Reply:
You are right, this file is not used in the project. Removed it from filesystem